db/crud_test.go

```diff
@@ -844,17 +844,20 @@ func BenchmarkDatabaseGet1xRev(b *testing.B) {
 
 	b.Run("ShortLatest", func(b *testing.B) {
 		for n := 0; n < b.N; n++ {
-			_, _ = db.Get1xRevBody("doc1", "", false, nil)
+			_, err := db.Get1xRevBody("doc1", "", false, nil)
+			require.NoError(b, err)
```

Comment:
As per previous comments - We should avoid assertions in benchmarks where possible - they introduce quite a lot of performance overhead that we don't want in a benchmark.

Reply:
Fixed in latest revision.